app/models/check.rb

```diff
@@ -53,4 +53,28 @@ def status
     return :caution if has_warnings?
     :ok
   end
+
+  def combined_errors
+    if link.monitor_link.present?
+      combine_link_history
+    else
+      link_errors
+    end
+  end
+
+private
+
+  def combine_link_history
```

Comment:
Might be nice to make this generic enough to work on the errors and the warnings.

Reply:
For now we're only worrying about errors, integration with warnings is on our backlog though